Documentation/users/tectonic-identity-overview.md

```diff
@@ -0,0 +1,58 @@
+# Tectonic Identity and user management
+
+User management in Tectonic is performed in two stages. The first stage authenticates a user, and the second authorizes the user to perform a given set of tasks associated with a role. Authenticating users in Tectonic is managed by Tectonic Identity, authorizing users is controlled by Kubernetes [Role-Based Access Control (RBAC)][link]. All Tectonic clusters enable RBAC which uses the user information produced by Tectonic Identity to enforce permissions.
+
+question: doesn't Tectonic Identity handle both auth and rbac? - beth
+
+Tectonic Identity is an authentication service for both Tectonic Console and `kubectl` and allows these components to talk to the API server on an end user's behalf. Users are either defined in the Tectonic `ConfigMap` or integrated with Tectonic through an external Identity Provider (IdP).
+
+This document provides an overview of Tectonic Identity, and its use in user authentication and access control.
+
+For more information see:
+* [Tectonic Identity configuration][identity-config]
+* [LDAP integration][ldap-integration]
+* [SAML integration][saml-integration]
+
+## Components of Tectonic Identity
+
+The three major components of Tectonic Identity are the API server, Tectonic Console, and kubectl.
+
+this doesn't make sense. these three aren't 'components' of TI - I think this means to list three things that 'access' TI. I also think this section should refer to 'Tectonic Identity', and not 'Dex' when describing these connections. - beth
```

Comment:
I think this section could probably just use a diagram that we then break down.

* Tectonic Identity - logs users into cooperate identity systems (LDAP, SAML, etc.) and issues credentials.
* Kubernetes API server - will authenticate user credentials against Tectonic Identity.
* Tectonic Console - requests credentials on the user's behalf from Tectonic Identity.
* kubectl - users can download credentials from Tectonic Console to use with kubectl when issuing commands to the Kubernetes API server.